### PR TITLE
Add AmazonMQ Broker W/Out EncryOptions query for CloudFormation #1036

### LINKED ISSUE #1036: Add AmazonMQ Broker Without EncryptionOptions query for AWS CloudFormation
### Platform
CloudFormation

### Provider
AWS

### Description
AmazonMQ Broker should be Encryption Options defined

### PR DESCRIPTION
**Pull request by joelcarvalhocheckmarx**
AmazonMQ Broker should be Encryption Options defined

Closes #1036

### CODE REVIEW

File: assets/queries/cloudFormation/amazonMQ_broker_without_encryption_options/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "amazonMQ_broker_without_encryption_options",
+  "queryName": "AmazonMQ Broker Without EncryptionOptions",
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/amazonMQ_broker_without_encryption_options/metadata.json:3]: change query id to `amazonmq_broker_encryption_disabled`

also change the query directory name to: `amazonmq_broker_encryption_disabled`

queryName = `AmazonMQ broker encryption disabled`

File: assets/queries/cloudFormation/amazonMQ_broker_without_encryption_options/metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "severity": "MEDIUM",
+  "category": null,
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/amazonMQ_broker_without_encryption_options/metadata.json:5]: `"category": "Encryption and Key Management",`

File: assets/queries/cloudFormation/amazonMQ_broker_without_encryption_options/query.rego [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::AmazonMQ::Broker"
+  properties := resource.Properties
+  exists_eo := object.get(properties, "EncryptionOptions", "undefined") != "undefined"
+  not exists_eo
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/amazonMQ_broker_without_encryption_options/query.rego:9]: simplify this changing line 8 and 9 to:
`object.get(properties, "EncryptionOptions", "undefined") == "undefined"`

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
Almost good

**Comment by rogeriopeixotocx** [comment]
LGTM ✅

**Review by joelsou5a — APPROVED** [review]

**Review by joelsou5a — DISMISSED** [review]

**Pull request merged**